src/commands/update/datacenter.ts

```diff
@@ -69,23 +68,23 @@ async function update_datacenter_action(options: UpdateDatacenterOptions, name:
       });
     }
 
-    return newPipeline
-      .apply({
-        providerStore: command_helper.providerStore,
-        logger: logger,
-      })
-      .then(async () => {
-        await command_helper.saveDatacenter(name, newDatacenter, newPipeline);
-        command_helper.renderPipeline(newPipeline, { clear: !options.verbose });
-        clearInterval(interval);
-        console.log('Datacenter updated successfully');
-      })
-      .catch(async (err) => {
-        await command_helper.saveDatacenter(name, newDatacenter, newPipeline);
-        clearInterval(interval);
-        console.error(err);
-        Deno.exit(1);
-      });
+    await command_helper.applyDatacenter(name, newDatacenter, newPipeline, logger);
+
+    command_helper.renderPipeline(newPipeline, { clear: !options.verbose });
+    if (interval) {
+      clearInterval(interval);
+    }
+    await sleep(1);
```

Comment:
What's the point of this?

Reply:
So for things like local the UI updates so fast that it is kind of triggering. This gives the UI a second so it does not look like random flashing,